testsuites/listener/shared/client_sg/conftest.py

```diff
@@ -125,7 +126,10 @@ def setup_client_syncgateway_test(request, setup_client_syncgateway_suite):
     try:
         client.delete_databases(ls_url)
         liteserv.stop()
-    except:
+    except HTTPError as h:
         liteserv.stop()
         logging_helper.fetch_and_analyze_logs(cluster_config=cluster_config, test_name=test_name)
-        raise Exception(sys.exc_info()[0])
+        raise HTTPError(h)
```

Comment:
Is it ok to wrap an exception in an exception? Why not just print a message in the exception that you raise?